bin/metrics-rds.rb

```diff
@@ -57,6 +57,13 @@ class RDSMetrics < Sensu::Plugin::Metric::CLI::Graphite
          proc:        proc { |a| Time.parse a },
          description: 'CloudWatch metric statistics end time'
 
+  option :fetch_age,
```

Comment:
If we want to stay consistent:

* ``--fetch-age`` should be ``--fetch_age`` but that's awful
* default should be 60 like the other places where ``fetch_age`` is set, but in that case it breaks the compatibility. 

@majormoses Opinion?

Reply:
So I would say name it `--fetch-age` and we can look at changing them all to be consistent in the future. For the threshold lets avoid the breaking change and we can update that in a future PR. I am not against breaking changes, its all about how how you do it and communicate the changes.